spec/exception_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
issue: Heh I'm just realizing something, we have `errors` in the `exception` namespace. They should probably all be `Error` instead of `Exception`, that's true also of the name of this file.

Reply:
I mean, they should have been called `Exceptions` from the beginning... :wink: